.github/workflows/gorgone.yml

```diff
@@ -14,7 +14,7 @@ on:
       - ready_for_review
     paths:
       - "gorgone/**"
-      - "!gorgone/tests/**"
+      - "gorgone/tests/**"
```

Comment:
do not forget to revert this change, as it may trigger an unexpected release if only tests are modified during a release.